app-sso/getting-started/provision-auth-server.hbs.md

```diff
@@ -24,209 +25,75 @@ tanzu package installed list -A | grep "sso.apps.tanzu.vmware.com"
+
+```bash
+   tanzu services classes list
+```
+
+Assuming there isn't already a Login Offering you want to connect to, you will want to create your own. 
 
 > **Caution** This `AuthServer` example uses an unsafe testing-only identity provider. Never use it in
 production environments. For more information about identity providers, see [Identity providers](../how-to-guides/service-operators/identity-providers.md).
 
-```yaml
----
+## Set Up Your First `ClusterUnsafeTestLogin`
+
+In a non-poduction environment, [`ClusterUnsafeTestLogin`](../../reference/api/clusterunsafetestlogin.hbs.md) is the
+recommended way to get started with Application Single Sign-On (commonly called AppSSO). 
+
+```bash
+cat <<EOF | kubectl apply -f -
 apiVersion: sso.apps.tanzu.vmware.com/v1alpha1
-kind: AuthServer
-metadata:
-  name: my-authserver-example
-  namespace: default
-  labels:
-    name: my-first-auth-server
-    env: tutorial
-  annotations:
-    sso.apps.tanzu.vmware.com/allow-client-namespaces: "default"
-    sso.apps.tanzu.vmware.com/allow-unsafe-issuer-uri: ""
-    sso.apps.tanzu.vmware.com/allow-unsafe-identity-provider: ""
-spec:
-  replicas: 1
-  tls:
-    deactivated: true
-  identityProviders:
-    - name: "internal"
-      internalUnsafe:
-        users:
-          - username: "user"
-            password: "password"
-            email: "user@example.com"
-            emailVerified: true
-            roles:
-              - "user"
-  tokenSignature:
-    signAndVerifyKeyRef:
-      name: "authserver-signing-key"
-
----
-apiVersion: secretgen.k14s.io/v1alpha1
-kind: RSAKey
+kind: ClusterUnsafeTestLogin
 metadata:
-  name: authserver-signing-key
-  namespace: default
-spec:
-  secretTemplate:
-    type: Opaque
-    stringData:
-      key.pem: $(privateKey)
-      pub.pem: $(publicKey)
+  name: my-unsafe-test-login-example
+EOF
 ```
 
-You can wait for the `AuthServer` to become ready with:
-
-```shell
-kubectl wait --for=condition=Ready authserver my-authserver-example
-```
+## Verify that your AuthServer is running
 
-Alternatively, you can inspect your `AuthServer` like any other resource:
+You can now see the service offering with:
 
 ```shell
-kubectl get authservers.sso.apps.tanzu.vmware.com --all-namespaces
+tanzu services classes list
 ```
 
-and you should see:
+You should see
 
 ```shell
-NAMESPACE NAME                  REPLICAS ISSUER URI                                         CLIENTS STATUS
-default   my-authserver-example 1        http://my-authserver-example.default.<your domain> 0       Ready
+NAME                          DESCRIPTION
+my-unsafe-test-login-example  Login by AppSSO - user:password - UNSAFE FOR PRODUCTION!
 ```
 
-As you can see your `AuthServer` gets an issuer URI templated. This is its entrypoint. You can find an `AuthServer`'s
-issuer URI in its status:
+__Note:__ As you can see this Login offering is *not* safe for production as it hard codes a user and password, but
+it is the quickest way to get started.
```

Comment:
**[suggestion]:** This note should be using the [recommended markup for notes](https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=CSOT&title=Using+DocWorks+Markdown#UsingDocWorksMarkdown-notesUsingNotes).